src/page-response-impact/RtSummaryStats.tsx

```diff
@@ -0,0 +1,157 @@
+import numeral from "numeral";
+import React from "react";
+import { ResponsiveOrdinalFrame } from "semiotic";
+import styled from "styled-components";
+
+import Colors from "../design-system/Colors";
+import { RtData, RtRecord } from "../infection-model/rt";
+import { RtDataMapping } from "../page-multi-facility/FacilityContext";
+import * as rtStats from "./rtStatistics";
+
+interface Props {
+  rtData: RtDataMapping;
+}
+
+const RtSummaryStatsContainer = styled.div`
+  display: flex;
+  flex-flow: row wrap;
+
+  > .responsive-container {
+    flex: 1 1;
+  }
+`;
+const RtSummaryTextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex: 1 1;
+  justify-content: flex-end;
+`;
+
+const LegendContainer = styled.div`
+  display: flex;
+  color: ${Colors.opacityForest};
+  flex-flow: row nowrap;
+  font-family: "Poppins", sans serif;
+  font-size: 9px;
+  font-weight: normal;
+  line-height: 16px;
+`;
+
+const LegendSpan = styled.span`
+  align-self: center;
+  background-color: #447f7c;
+  height: 10px;
+  margin-right: 8px;
+  width: 26px;
+`;
+
+const RtSummaryText = styled.div`
+  color: ${Colors.opacityForest};
+  font-family: "Helvetica Neue", sans serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: normal;
+  line-height: 150%;
+`;
+
+function filterRtData(data: RtData | null): data is RtData {
+  return data !== null;
+}
+
+const RtSummaryStats: React.FC<Props> = ({ rtData }) => {
+  const rtDataValues: (RtData | null)[] = Object.values(rtData);
+  const totalFacilities = rtDataValues.length;
+
+  const facilitiesRtRecords: RtRecord[][] = rtDataValues
+    .filter(filterRtData)
+    .map((rtData: RtData) => rtData.Rt);
+
+  const numFacilitiesWithRtLt1 =
+    rtStats.numFacilitiesWithRtLessThan1(facilitiesRtRecords) || 0;
+
+  const averageRtReductionAcrossFacilities =
+    rtStats.averageRtReductionAcrossFacilities(facilitiesRtRecords) || 0;
+
+  const numFacilitiesWithRtGt1 = totalFacilities - numFacilitiesWithRtLt1;
```

Comment:
shouldn't this be subtracted from `facilitiesRtRecords.length` instead? here we are including facilities where we don't actually know what the Rt is.

Reply:
Oh okay yeah -- I think I misunderstood a conversation with @sychang about this. I'll check with her real quick to confirm it should be the total number of facilities we have Rt data for.

Reply:
Okay so from @sychang:
"X will be the # of facilities 1) that we have data for 2) that have rt <1
Y is the total number of facilities that they have modelled overall, not just the number we have rt values for"

So I think that means we should keep it as is with `rtDataValues.length` (which I'm assuming would have a value for every facility modeled, even if some return null)

Reply:
yeah that makes sense ... in that case I think you should rename this variable to avoid future confusion about what this is supposed to represent